src/app/pages/send/ordinal-inscription/coinselect/select-inscription-coins.spec.ts

```diff
@@ -37,11 +37,11 @@ describe(selectInscriptionTransferCoins.name, () => {
         Number(inscriptionInputAmount)
     );
 
-    expect(result.txFee).toEqual(6765);
+    expect(result.txFee).toEqual(5048);
```

Comment:
How is this value of `5048` decided? Maybe we can give it a `const` name for readability?

Reply:
since I will refactor fees gen, will return to this test in future pr. thanks Pete!